test/command/pinning.spec.ts

```diff
@@ -112,7 +112,7 @@ describe('Test Pinning command', () => {
     expect(consoleMessages[3]).toContain('Reuploaded successfully.')
   })
 
-  it('should allow reuploading pinned folder', async () => {
+  test.skip('should allow reuploading pinned folder', async () => {
```

Comment:
have we created an issue for the flaky reupload problem in the Bee repo? if so, please place a comment above this line with the issue link, starting with `FIXME` or create issue about it.